apps/src/templates/policy_compliance/NewStudentAccountConsent.story.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
```

Comment:
I've heard mixed messaging around whether or not we're supposed to make new storybooks. It seems like a useful thing to me, but does it take the place of unit tests for the component?

Reply:
I thought storybooks are not a replacement for unit tests. They are just a way to quickly make changes to the component since you don't need to run the whole website. 

I should add unit tests for this component though.

Reply:
Who is telling you that we're not supposed to make new storybooks? We just spent what feels like a year updating them. :)

I should also add unit tests hmm.

Reply:
@wilkie it was actually the React onboarding presentation :joy:

Reply:
🙃

Reply:
Unit tests added.